model.js

```diff
@@ -0,0 +1,117 @@
+const fs = require('fs')
+
+class Model {  
+    constructor() {
+        this.data = this.readData()
+    }
+
+    readData() {return JSON.parse(fs.readFileSync('data.json', 'utf8'))}
+
```

Comment:
setelah melakukan JSON.parse, data seharusnya diubah dari array of object menjadi array of instance model, baru kemudian di-return